index.bs

```diff
@@ -339,13 +342,66 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 :: A user agent implementing, in conjunction with the underlying platform, the [=Web Authentication API=] and algorithms
     given in this specification, and handling communication between [=authenticators=] and [=[RPS]=].
 
+: <dfn>Credential ID</dfn>
+:: A probabilistically-unique [=byte sequence=] identifying a [=public key credential source=] and its [=authentication assertions=].
+
+    Credential IDs are generated by [=authenticators=] in two forms:
+    1. At least 16 bytes that include at least 100 bits of entropy, or
+    1. The [=public key credential source=], without its [=Credential ID=], encrypted so only the owning authenticator can
```

Comment:
There could also be a third form: some hash value derived from the public key or some encrypted data.  We might want to rephrase the option 1 to include such approach.

Reply:
Or do you consider such option being included in 1. already?

Reply:
@equalsJeffH I've added a note referring to those guidelines for now.

@rlin1 I changed item 1 in f1f9a07f6ac17456d059f56b60dcfbef3bb4ce79 to talk about bits of entropy instead of random numbers in order to include hashes.